### ISSUE TITLE
P-value as score

### ISSUE THREAD

**Issue by iamjli** [jhkorhonen/MOODS#18]
Hi quick question, just wondering if it's possible to output p-value as column for motif matches. Right now I'm only seeing LOD score. Or is the best way to do this just to calculate a bunch of different thresholds for various p-values? 

Thanks!

**Comment by jhkorhonen** [comment]
No. Computing the p-value from the score is unfortunately not very straightforward, so it would need some effort to do efficiently.

(In principle, it's implemented in the p-value from threshold code, but it would need to be tweaked to get all desired p-values out in a one go in some sane way.)
